src/com/epam/javacourse/hometask3/Bookstore.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.javacourse.hometask3;
+
+public abstract class Bookstore {
+    public static void displayBooksByAuthor(String authorName, Book[] books) {
+        for (Book book : books) {
+            for (Author author : book.getAuthors()) {
+                if (author.getFullName().equals(authorName)) {
```

Comment:
And you can use an override `equals()` method to compare whole `Author` objects.
This is an example https://www.baeldung.com/java-comparing-objects#equals-instance.